charts/argo-events/templates/eventsource-controller-deployment.yaml

```diff
@@ -72,3 +72,7 @@ spec:
       {{- if .Values.eventsourceController.affinity }}
       affinity: {{ toYaml .Values.eventsourceController.affinity | nindent 8 }}
       {{- end }}
+      {{- if .Values.eventsourceController.resources }}
+      resources:
```

Comment:
`resources` is not a pod-level setting. Please move this inside the `containers` section.